### DISCUSSION TITLE
How to type an acute accent (◌́)

### DISCUSSION THREAD

**Discussion by mamekoro** [google/mozc#896] [Q&A] [answered]
How do I type a character with an acute accent like "é" in "Pokémon"?

**Comment by shitamo** [comment]
https://dekiru.net/article/14413/

**Reply by mamekoro** [reply, marked as answer]
Thanks. It seems the article was written for Microsoft IME and iPhone, but it was helpful.

I managed to type "é" by following these steps.
1. Activate Mozc IME
2. Press Shift+無変換 to enable 半角英数 mode
3. Enter "e"
4. Enter Space or 変換 and select "é"
